Change default CE pin from 16 to 4 (#371)
sidoh/esp8266_milight_hub: README.md

##### NRF24L01+

[This guide](https://www.mysensors.org/build/connect_radio#nrf24l01+-&-esp8266) details how to connect an NRF24 to an ESP8266. By default GPIO 4 for CE and GPIO 15 for CSN are used, but these can be configured later.

##### LT8900

> Connect SPI pins (CS, SCK, MOSI, MISO) to appropriate SPI pins on the ESP8266. With default settings, connect RST to GPIO 0, and PKT to GPIO 16. By default GPIO 4 for CE and GPIO 15 for CSN are used, make sure to properly configure these.

Reviewer: Small language suggestion for clarity:

```suggestion
Connect SPI pins (CE, SCK, MOSI, MISO) to appropriate SPI pins on the ESP8266. With default settings, connect RST to GPIO 0, PKT to GPIO 16, CE to GPIO 4, and CSN to GPIO 15.  Make sure to properly configure these if using non-default pinouts.
```

Also, I believe `CS` and `CE` are interchangable, but I tend to use CE.  I might as well attempt to be consistent. :)
